k8s/README.md

```diff
@@ -48,54 +48,33 @@ The production branch maintains an image that can be released in production with
     -   ./regions/oregon-b/stage-secrets.yaml
     -   ./regions/oregon-b/prod-secrets.yaml
```

Comment:
The kops cluster had a specific way of handling secrets. Will this be preserved in `eks`?

Reply:
Probably this section needs updating too